themes/LaserWave-color-theme.json

```diff
@@ -1,18 +1,20 @@
 {
     "name": "LaserWave",
     "colors": {
+        "focusBorder": "#40b4c4",
```

Comment:
I like having the focus border be different, but I think going from pink to blue (on the inputs) is a bit too stark of a jump. I would make this focusBorder the default pink (`#EB64B9`) and updating the `input.border` to a darker version of the pink (Something in the range of `#964c7b`) so that it goes from a dim pink to a bright pink

Reply:
Fixed in https://github.com/Jaredk3nt/laserwave/pull/19/commits/4d7ce2a810531500ae71e4a502d1f02203d3b11b